tasks/components/CloseOrderTask.js

```diff
@@ -0,0 +1,14 @@
+import h from 'react-hyperscript'
+import { isNil } from 'ramda'
+
+export default (props) => {
+  const { taskPlan, currentOrder } = props
+  if (isNil(taskPlan)) return null
+  if (isNil(currentOrder)) return null
+  const { params: { orderId } } = taskPlan
+  const { consumerAgentId, supplierAgentId } = currentOrder
+
+  return h('div', {}, [
+    h('h2', {}, `Closing order ${orderId} for group ${consumerAgentId} from supplier ${supplierAgentId}`)
```

Comment:
shouldn't this be internationalized?

Reply:
it could be, but it's only temporary text until we actually have a way to properly close an order... feels like it'd just be more code to remove later on.